js/stores/windowStore.js

```diff
@@ -431,6 +431,9 @@ const doAction = (action) => {
     case WindowConstants.WINDOW_NEW_FRAME:
       newFrame(action.frameOpts, action.openInForeground)
       break
+    case WindowConstants.WINDOW_VIEW_KEY:
+      newFrame(action.frameOpts, action.openInForeground)
```

Comment:
I think I like the idea of having a separate action for this, but there's not much point in having a separate action if all it does is call newFrame. I would suggest a few things:
1. This can be moved to the appStore now that we have a good way to open new tabs there
2. The handler for the action that opens the new window should know what the url and other options are